xbmc/interfaces/python/PythonSwig.cpp.template

```diff
@@ -117,6 +126,8 @@ void doMethod(Node method, MethodType methodType)
     else
       return;
   }
+  else if (method.@name == "contains")
```

Comment:
Hi @Montellese,

This seems like an odd special case. If my method name is exactly "contains" then I'm going to generate the method call differently?